core/src/main/res/values/strings.xml

```diff
@@ -527,6 +527,12 @@
     <string name="back_button_go_to_page_title">Select page</string>
     <string name="pref_delete_removes_from_queue_title">Delete removes from Queue</string>
     <string name="pref_delete_removes_from_queue_sum">Automatically remove an episode from the queue when it is deleted.</string>
+    <string name="pref_filter_feed_title">Subscription Filter</string>
+    <string name="pref_filter_feed_sum">Filter your subscriptions screen</string>
```

Comment:
What about "Filter your subscriptions in navigation drawer and subscriptions screen"?